requirements.txt

```diff
@@ -24,7 +24,7 @@ swagger-spec-validator==2.4.3
 marshmallow==3.9.0
 treelib==1.5.5
 Twisted[tls]==20.3.0
-typing==3.7.4.1
+typing==3.7.4.1; python_version<'3.7'
```

Comment:
I'd argue that this is not needed because the change in `setup.py` takes care of it. The `Dockerfile` uses `pip install ./` which calls `setup.py` as well.